blockchain/contracts/FlatNFTtheWorld.sol

```diff
@@ -1264,7 +1264,7 @@ contract NFTtheWorld {
 
     mapping(address => mapping(uint256 => string[]))
         private nftReservationInformationOfUsers;
-    mapping(address => uint256[]) private nftAssetsInformationOfUsers;
```

Comment:
Please get rid of `nftAssetsInformationOfUsers[msg.sender].push(nftToken);` in `buyNFT` then as well :)